src/main/scala/strawman/collection/mutable/ArrayDeque.scala

```diff
@@ -0,0 +1,292 @@
+package scala
+package collection.mutable
```

Comment:
@julienrf : But I need access to `.sizeHintIfCheap` which is only package visible to `scala.collection.mutable`. This is not a hard dependency but it lets me special case some operations when size can be known before hand to pre-allocate the array.